package.json

```diff
@@ -27,6 +27,7 @@
     "deploy": "ember deploy production",
     "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
     "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:glint": "glint",
```

Comment:
**TODO**

Update `tsconfig.json` so that `lint:types` runs `glint`. (We may need a separate script to type-check Node files.)